srcpkgs/Signal-Desktop/template

```diff
@@ -1,38 +1,34 @@
 # Template file for 'Signal-Desktop'
 pkgname=Signal-Desktop
-version=6.20.0
+version=6.20.1
 revision=1
 # Signal officially only supports x86_64 
 # x86_64-musl could potentially work based on the Alpine port:
 # https://git.alpinelinux.org/aports/tree/testing/signal-desktop
 # ARM: https://github.com/signalapp/Signal-Desktop/issues/3410
 archs="x86_64"
-hostmakedepends="git git-lfs nodeenv python3 tar"
+hostmakedepends="git git-lfs nodejs python3 tar"
 depends="cairo gtk+3 libvips pango"
 short_desc="Signal Private Messenger for Linux"
 maintainer="anelki <akierig@fastmail.de>"
 license="AGPL-3.0-only"
 homepage="https://github.com/signalapp/Signal-Desktop"
 distfiles="https://github.com/signalapp/Signal-Desktop/archive/v${version}.tar.gz"
-checksum=fc92011c51da6792215eb92d48afc17f1967caeade1307daaa13884244d7a3b0
+checksum=3c664dbb5c1670554859fa55fd765ac35632f573fd5b7bea63876cdf41b6b7d6
 nostrip_files="signal-desktop"
 
 post_extract() {
 	# git-lfs hook needs to be installed for one of the dependencies
 	git lfs install
 
-	# use nodeenv since we have that in the repos already
-	nodeenv --node='18.14.0' signal-node
-	. ./signal-node/bin/activate
+	vsed 's/"node": "/&>=/' -i package.json
 
 	npm install -g yarn
```

Comment:
is this still necessary?

Reply:
yes, our node is 18.16, the package.json file specifies 18.14 specifically. 

https://github.com/signalapp/Signal-Desktop/blob/7ec64823bdce91f2e6802566dfe9963f5b9b5363/package.json#L320